### PR TITLE
Clean up code related to V1

### PR DESCRIPTION
**Pull request by Vidya2606**
To Cleanup and complete Binary Utilities V1 deprecation work.

### CODE REVIEW

**Review by elenavillamil — COMMENTED** [review]
Hi Vidya, thanks for the change :) I think IBinariesUtility can be cleaned up too.

File: src/ExtensionRoot.ts [hunk 1]
```diff
@@ -94,7 +93,7 @@ export class ExtensionRoot {
         const experimentationService = await createExperimentationServiceAsync(context, new ExperimentationTelemetry(this._logger));
 
         const expectedCLIVersion: string = await VersionUtility.getExpectedCliVersionAsync(packageJsonContent);
-        const binariesUtility: IBinariesUtility = BinariesManager.getBinariesUtility(this._logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion);
```
Comment (Tatsinnit) [inline src/ExtensionRoot.ts:97]: 💡 Thank you for this, I believe you have tested this locally, so we are getting rid of one aka old `BinariesManager.getBinariesUtility` with choice to `v2` ? For rest of deleted files have we made sure there is not other file referencing this funct? please. thanks.

**Review by Tatsinnit — APPROVED** [review]
☕️💡 Thank you for adding me into this review, really appreciate it, given the changes look fairly straight forward might be please verify that none of the functionality from delete files are used anywhere else please?

Please make sure we have tested tshie by generating `six` sharing through the testing channels in place before merging this. ❤️🙏🥷

File: src/ExtensionRoot.ts (hunk 1 above)
Comment (Vidya2606) [reply src/ExtensionRoot.ts:97]: Thanks for the review, yes I have tested locally and made sure non of the files are referencing this func.

**Review by elenavillamil — APPROVED** [review]

**Pull request merged**